src/presentational/SectionTeaserContainer.tsx

```diff
@@ -6,7 +6,7 @@ import globalStyles, {hitSlop, standardPadding} from 'src/styles';
 type PropTypes = {
   title: string;
   onMorePress: () => void;
-  children: React.ReactNode;
+  children?: React.ReactNode;
```

Comment:
Why making this optional?

Reply:
for referenda I didn't add anything as body yet just made it a section header so we can navigate to.
<img src=https://user-images.githubusercontent.com/1168983/125089072-56424300-e0ce-11eb-984b-b28d4f87eb9f.png width=250 />

Reply:
I see, but as this component is a "container" the children should be mandatory.. You can pass a `Text` with and empty string as body or something like this until we have something to show or export the `Header` if it's the only thing we need.

Reply:
fdf228d
done ✅